libs/httpproxy/proxy-camel-servlet/src/main/java/org/codice/proxy/http/HttpProxyCamelHttpTransportServlet.java

```diff
@@ -38,6 +38,19 @@
  * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
  * express or implied. See the License for the specific language governing permissions and
  * limitations under the License.
+ *
+ * <p>Licensed to the Apache Software Foundation (ASF) under one or more contributor license
```

Comment:
Why was this added? Looks like we have it duplicated already. Should we be removing one instead of adding one.

Reply:
@clockard This must have gotten duplicated when I did the formatter, I'll remove the duplicates